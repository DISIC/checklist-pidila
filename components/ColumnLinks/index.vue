<template>

  <div class="secondary-nav-box">
    <h2 class="nav-box-title">Les autres checklists</h2>
    <div class="nav-box-content">
      <ul>
        <li><a target="_blank" title="nouvelle fenêtre" href="http://pidila.gitlab.io/amoa/evaluation-rapide.html">Évaluation rapide</a> (avant de faire un audit)</li>
        <li><a target="_blank" title="nouvelle fenêtre" href="http://pidila.gitlab.io/developpement/shortlist.html">Checklist dev</a> (avant de livrer)</li>
      </ul>
    </div>
<!--
    <h2 class="nav-box-title">Checklist Pidila</h2>
    <div class="nav-box-content">
      <ul>
        <li><a target="_blank" title="nouvelle fenêtre" href="https://pidila.gitlab.io/checklist/a-propos-de-la-checklist.html">À propos de la
          checklist</a></li>
        <li><a target="_blank" title="nouvelle fenêtre" href="https://pidila.gitlab.io/checklist/referentiels.html">Référentiels inclus</a></li>
        <li><a target="_blank" title="nouvelle fenêtre" href="https://pidila.gitlab.io/checklist/actualisations.html">Actualisations</a></li>
      </ul>
    </div>
-->    
  </div>
</template>

<script>
  export default {
    name: 'ColumnLinks'
  }
</script>
