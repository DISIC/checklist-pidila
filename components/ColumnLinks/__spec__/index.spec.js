import {shallowMount} from '@vue/test-utils'

import ColumnLinks from '../index.vue'

describe('components | ColumnLinks', () => {

  it('should match snapshot', () => {
    // when
    const wrapper = shallowMount(ColumnLinks)

    // then
    expect(wrapper.element).toMatchSnapshot()
  })

})
