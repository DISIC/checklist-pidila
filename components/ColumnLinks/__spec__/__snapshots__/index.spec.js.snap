// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | ColumnLinks should match snapshot 1`] = `
<div
  class="secondary-nav-box"
>
  <h2
    class="nav-box-title"
  >
    Checklist Pidila
  </h2>
   
  <div
    class="nav-box-content"
  >
    <ul>
      <li>
        <a
          href="https://pidila.gitlab.io/checklist-pidila/a-propos-de-la-checklist.html"
        >
          À propos de la
        checklist
        </a>
      </li>
       
      <li>
        <a
          href="https://pidila.gitlab.io/checklist-pidila/referentiels.html"
        >
          Référentiels inclus
        </a>
      </li>
       
      <li>
        <a
          href="https://pidila.gitlab.io/checklist-pidila/actualisations.html"
        >
          Actualisations
        </a>
      </li>
    </ul>
  </div>
</div>
`;
